scss/_base_icon-definitions.scss

```diff
@@ -172,3 +172,353 @@ $social-icon-size: map-get($icon-sizes, heading-icon--small);
+}
+
+@mixin vf-icon-show($color) {
+  background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M8 3.002c2.946 0 5.612 1.666 8 4.998-2.388 3.332-5.054 4.998-8 4.998-2.946 0-5.612-1.666-8-4.998 2.388-3.332 5.054-4.998 8-4.998zM8 5a3 3 0 100 6 3 3 0 000-6zm0 1.49a1.51 1.51 0 110 3.02 1.51 1.51 0 010-3.02z' fill='#{vf-url-friendly-color($color)}'  fill-rule='evenodd'/%3E%3C/svg%3E");
+}
+
+@mixin vf-icon-sort-both($color) {
+  background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cg fill='#{vf-url-friendly-color($color)}'  fill-rule='nonzero'%3E%3Cpath d='M8 2l3.333 5H4.667zM8 14L4.667 9h6.666z'/%3E%3C/g%3E%3C/svg%3E");
+}
+
+@mixin vf-icon-sort-down($color) {
+  background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='#{vf-url-friendly-color($color)}'  fill-rule='nonzero' d='M8 14L4.667 9h6.666z'/%3E%3C/svg%3E");
+}
+
+@mixin vf-icon-sort-up($color) {
+  background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='#{vf-url-friendly-color($color)}'  fill-rule='nonzero' d='M8 2l3.333 5H4.667z'/%3E%3C/svg%3E");
+}
+
+@mixin vf-icon-starred($color) {
+  background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='#{vf-url-friendly-color($color)}'  d='M8 13l-4.702 2.627.898-5.564-3.804-3.94 5.257-.811L8 .25l2.351 5.062 5.257.811-3.804 3.94.898 5.564z' fill-rule='evenodd'/%3E%3C/svg%3E");
+}
+
+@mixin vf-icon-status-failed-small($color) {
```

Comment:
I believe all `status-X-small` icons are exactly the same SVG with different color, they could use one definition mixin (`vf-icon-status-small()`) that would define the SVG, and this can be either used directly in pattern mixins (that would define different colors) or we can have icon specific mixins that just calls the SVG mixin with correct color:

```
@mixin vf-icon-status-failed-small($color: $correct-color-for-warning-icon) {
   @include vf-icon-status-small($color);
}
```

This would help remove a bit on unnecessary duplication of SVGs.